workers/repomirrorworker/test/test_repomirrorworker.py

```
@@ -1533,3 +1535,235 @@ def test_push_sparse_manifest_list_string_manifest(
     call_kwargs = mock_put.call_args[1]
     assert isinstance(call_kwargs["data"], bytes)
     assert call_kwargs["data"] == manifest_str.encode("utf-8")
+
+
+# =============================================================================
+# Tests for process_mirrors() edge cases
+# =============================================================================
+
+
+@mock.patch("workers.repomirrorworker.features")
+def test_process_mirrors_feature_disabled(mock_features, initialized_db, app):
```

Ruff (0.14.14)

[warning] Unused function argument: `initialized_db`

(ARG001)